fedimint-cli/src/lib.rs

```diff
@@ -720,11 +719,7 @@ impl FedimintCli {
                         .await
                         .map_err_cli_general()?,
                     None => ws_api
-                        .request_with_strategy(
-                            ThresholdConsensus::full_participation(ws_api.peers().len()),
-                            method,
-                            params,
-                        )
+                        .request_current_consensus(method, params)
```

Comment:
Was this actually a good idea? It's only a dev command. It didn't occur to me when I looked at this PR, but now I'm reviewing #4220 and I'm wondering if we sometimes actually want to wait for all the guardians in tests? @joschisan @dpc

Reply:
I don't recall any case like that, and if they do/will need it, they can loop over each peer_id etc.

Reply:
Problem was that it was not marked as such and caused confusion when we took one peer offline. Just using loops for now if needed at all seems fine.